microcosm_flask/tests/conventions/test_discovery.py

```diff
@@ -0,0 +1,41 @@
+"""
+Test the discovery endpoint.
+
+"""
+from json import loads
+
+from hamcrest import (
+    assert_that,
+    equal_to,
+    is_,
+)
+
+from microcosm.api import create_object_graph
+from microcosm_flask.operations import Operation
+
+
+def test_discovery():
+    graph = create_object_graph(name="example", testing=True)
+    # TODO: can we make this automatic?
```

Comment:
I ran into a design flaw here:
 - This code only references `route`, which only references the root `flask` app, not the fully configured `app` that includes `discovery`, `health`, etc.
 - I can't easily change `route` to reference `app` because it creates a circular dependency
 - If just using `route` is a common use pattern, we force users to call `graph.use()` on boilerplate that would otherwise be called when initialization `app`

Need to think about this more.